Documentation/ApiOverview/CachingFramework/QuickStart/Index.rst

```diff
@@ -22,7 +22,7 @@ and can be overridden in :file:`LocalConfiguration.php`.
 
 If specific settings should be applied to the configuration, they should be added to :file:`LocalConfiguration.php`.
 All settings in :file:`LocalConfiguration.php` will be merged with :file:`DefaultConfiguration.php`. The easiest way to see
-the final cache configuration is to use the TYPO3 Backend module Admin Tools > Configuration > :php:`$GLOBALS['TYPO3_CONF_VARS']`.
+the final cache configuration is to use the TYPO3 Backend module **ADMIN TOOLS > Configuration > :php:`$GLOBALS['TYPO3_CONF_VARS']`**.
```

Comment:
In this specific case I'd even remove the syntax highlighting since this is a description of a UI label which does not have syntax highlighting in reality too. ;-)